check/controls.go

```diff
@@ -49,7 +51,25 @@ func NewControls(in []byte) (*Controls, error) {
 	// Prepare audit commands
 	for _, group := range c.Groups {
 		for _, check := range group.Checks {
-			check.Commands = textToCommand(check.Audit)
+			if check.SubChecks == nil {
+				check.Commands = textToCommand(check.Audit)
+			} else {
+				for i, SubCheck := range check.SubChecks {
+					check.SubChecks[i].Commands = textToCommand(SubCheck.Audit)
+				}
+			}
+		}
+	}
+
+	if len(definitions) > 0 {
+		c.DefinedConstraints = map[string][]string{}
+		for _, val := range definitions {
+			a := strings.Split(val, "=")
+
+			// If its type 'category=value' for example 'platform=ubuntu'
+			if len(a) == 2 {
```

Comment:
Might also need to trim spaces e.g. for `--define foo = bar` (or would we reject that somewhere?)

Reply:
in that case we would have just "foo" without the "= bar" so we can't do anything about that and it will be rejected in here